kubernetes/deployment.yaml

```diff
@@ -20,28 +20,23 @@ spec:
         ports:
         - containerPort: 3000
         env:
-          - name: ASTRA_APPLICATION_TOKEN
-            valueFrom:
-              secretKeyRef:
-                name: astra-application-token
-                key: ASTRA_APPLICATION_TOKEN
-          - name: ASTRA_DATABASE_ID
+          - name: APPROVED_TOKENS
             valueFrom:
               secretKeyRef:
-                name: astra-database-id
-                key: ASTRA_DATABASE_ID
-          - name: ASTRA_DATABASE_REGION
+                name: approved-tokens
+                key: APPROVED_TOKENS
+          - name: API_MONGO_CONNECTION_STRING
             valueFrom:
               secretKeyRef:
-                name: astra-database-region
-                key: ASTRA_DATABASE_REGION
-          - name: APPROVED_TOKENS
+                name: api-mongo-connection-string
+                key: API_MONGO_CONNECTION_STRING
+          - name: API_MONGO_DB_NAME
             valueFrom:
               secretKeyRef:
-                name: approved-tokens
-                key: APPROVED_TOKENS
-          - name: JWT_SECRET
+                name: api-mongo-db-name
+                key: API_MONGO_DB_NAME
+          - name: GH_TOKEN
```

Comment:
@Cahllagerfeld I had to rename the envars so that they don't clash with other apps deployed to the same Kubernete cluster

Reply:
Not `GH_TOKEN` and `APPROVED_TOKENS` these can be global but the others I think we should prepend with the application name